tests/suite/sample_test.go

```diff
@@ -14,7 +16,8 @@ import (
 	"github.com/nginxinc/nginx-gateway-fabric/tests/framework"
 )
 
-var _ = Describe("Basic test example", func() {
+var _ = Describe("Basic test example", Ordered, Label("functional"), func() {
```

Comment:
This doesn't need to be Ordered, so you can leave the BeforeEach and AfterEach in place.

Reply:
sound good